doc/quick-start/qs-guide.rst

```diff
@@ -232,7 +232,149 @@ If all the steps were followed correctly, it will be possible to see that the KN
 
 ----------------------------------------------------------------
 
-KNoT Cloud SDK
---------------
+Creating an Application on KNoT Cloud
```

Comment:
I didn't understand why **represent**  would be more accurate since it isn't created yet at this time. Notice that the application we're running on the thing is different from the application we built on top of the cloud (it encapsulates a specific set of business rules).

Reply:
Please, notice that this section stops at downloading the credentials, not creating an app.

Reply:
Done.